app/js/components/validation.js

```diff
@@ -25,8 +25,9 @@ if (form.length) {
         const parent = $(this).closest('.form-row');
         const errorContainer = parent.parents('.form-row').find('.parsley-errors').first();
         const translatedValidationText = $(this).data('motivation-keep-talking');
+        const words = $(this).val().split(' ').filter(v => v !== "");
```

Comment:
The filter is a nice and elegant way of filtering away empty entries. Using `v` as the name for the function input is not very descriptive. Using `word` instead of `v` might be a bit more self-explaining.